compose/ui/ui/src/webCommonW3C/kotlin/androidx/compose/ui/window/ComposeWindow.js.kt

```diff
@@ -102,6 +104,19 @@ private class ComposeWindow(
         this.addEventListener(type, { event -> handler(event as T, layer.view) })
     }
 
+    private fun initMediaEventListener(handler: (Float) -> Unit) {
+        val contentScale = density.density
+        window.matchMedia("(resolution: ${contentScale}dppx)")
+            .addEventListener("change", { evt ->
+                evt as MediaQueryListEvent
+                if (!evt.matches) {
+                    handler(contentScale)
+                    initMediaEventListener(handler)
```

Comment:
Should `initMediaEventListener ` be called outside of `if .. then ..` ? So for any change event?
Otherwise we won't resubsribe in some cases due to `once = true`, right?